typescript/packages/lattice-client-react/vite.config.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'node:path';
+import react from '@vitejs/plugin-react';
```

Comment:
```suggestion
import path from 'node:path';

import react from '@vitejs/plugin-react';
```

Absolutely an incredibly small NIT but I like a little space between the platform imports, code-we-didnt-write, and code-we-did-write. Take it or leave it!